8_child_illness.do

```diff
@@ -88,15 +94,20 @@ c_illtreat
 		}
 		replace c_diarrhea_hmf = 0 if ca1 == 1	         // children with diarrhea in last 2 weeks
 		
-		if ~inlist(country_name,"KyrgyzRepublic2018") {		
+		if ~inlist(country_name,"KyrgyzRepublic2018","Tonga2019") {		
 			replace c_diarrhea_hmf = 1 if c_diarrhea_hmf == 0 & ca7d == 1 	// received Government recommended homemade fluid 
 			replace c_diarrhea_hmf = . if inlist(ca7d, 8, 9) 	// missing when Government recommended homemade fluid  variable is missing and none o	
 		}
 		if inlist(country_name,"KyrgyzRepublic2018") {		
 			replace c_diarrhea_hmf = 1 if c_diarrhea_hmf == 0 & (ca7d == 1 | ca7e == 1) 	// received Government recommended homemade fluid 
 			replace c_diarrhea_hmf = . if inlist(ca7d, 8, 9) | inlist(ca7e, 8, 9) 	// missing when Government recommended homemade fluid  variable is missing and none o	
 		}
+		if inlist(country_name,"Tonga2019") {
+			replace c_diarrhea_hmf = 1 if c_diarrhea_hmf == 0 & ca7b == 1
+			replace c_diarrhea_hmf = . if inlist(ca7b, 8, 9)	// missing when Government recommended homemade fluid  variable is missing and none o	
```

Comment:
I have not identified relevant infformation on fluid and treatment for Diarrhoea in Chapter 7 of Tonga's survey report... Typically, we could use the specific table to extract what kind of diarrhoea treatment should be considered non-traditional.